store/pg_engine.go

```diff
@@ -368,15 +357,36 @@ func (db *DB) migrate(ctx context.Context, d dbdriver.Driver, curVer *semver.Ver
 				Environment:           "", /* unused in execute migration */
 				Source:                dbdriver.LIBRARY,
 				Type:                  dbdriver.Migrate,
-				Description:           fmt.Sprintf("Migrate version %s server version %s with files %s.", version, db.serverVersion, strings.Join(baseNames, ", ")),
+				Description:           fmt.Sprintf("Migrate version %s server version %s with files %s.", version, serverVersion, strings.Join(baseNames, ", ")),
 			},
 			stmtBuf.String(),
 		); err != nil {
-			return fmt.Errorf("failed to migrate schema version %q, error: %v", version, err)
+			return semver.Version{}, fmt.Errorf("failed to migrate schema version %q, error: %v", version, err)
 		}
+		retVersion = version
 	}
-	db.l.Info("Completed database migration.")
-	return nil
+	l.Info("Completed database migration.")
```

Comment:
same here

Reply:
Done. Also, changed to printing the message only when there's migration.